src/style.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --star: url(./images/star.svg);
+  --active-star: url(./images/star-active.svg);
+}
+
+* {
+  margin: 0;
```

Comment:
Redundant to make margin: 0 for all selectors.
Make just where it necessary. In this case just for div.

Reply:
UPD: * just for body